third_party/libpg_query/src/postgres/include/catalog/pg_opclass.h

```diff
@@ -84,7 +84,7 @@ typedef FormData_pg_opclass *Form_pg_opclass;
 /* ----------------
  *		initial contents of pg_opclass
  *
- * Note: we hard-wire an OID only for a few entries that have to be explicitly
```

Comment:
No. You should not change anything in the "third_party" directory. Otherwise we will not know what got modified if we ever need to upgrade to a newer version.

Expunge any changes to those files in your commit history.